schemas.go

```diff
@@ -83,13 +83,28 @@ type Schema struct {
 	Block *SchemaBlock `json:"block,omitempty"`
 }
 
+type StringKind string
```

Comment:
A bikeshed, but maybe we could think of a more descriptive name for this type? `tfjson.StringKind` is not entirely too descriptive of the purpose here. Also, and a doc comment be added?

Reply:
Reasonable, I went with a generic name as that is the type name in the protocol: https://github.com/hashicorp/terraform/blob/master/docs/plugin-protocol/tfplugin5.2.proto#L73

But I can make it `DescriptionKind` or something if you prefer, just not sure it will be limited to Description's going forward.

Reply:
I'm for longer types, even if it's a bit wordy 😛 

Maybe `SchemaBlockDescriptionKind`? 😆

Reply:
Its used in attribute too, i'll go with `SchemaDescriptionKind`